Makefile

```diff
@@ -0,0 +1,8 @@
+run:
```

Comment:
seems easier to just run the docker commands instead of masking them behind a makefile